src/components/form/Form/FormField.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { memo } from 'react';
+import _ from 'lodash';
+
+import { Form, StrictFormFieldProps } from 'semantic-ui-react';
```

Comment:
Probably more correct would be to have:
```suggestion
import { Form } from 'semantic-ui-react';
import type { StrictFormFieldProps } from 'semantic-ui-react';
```